crates/bevy_time/src/time.rs

```diff
@@ -174,19 +176,22 @@ impl Time {
         self.last_update = Some(instant);
     }
 
-    /// Returns the [`Instant`] the app was started.
+    /// Returns the [`Instant`] this clock was created.
+    /// Usually represents when the app was started.
```

Comment:
Minor nit: rustdoc will merge lines without an empty line between them into a single paragraph, which can result in an overlong summary when looking at a list of items. [The rustdoc book](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components) recommends keeping summaries to a single line.

```suggestion
    /// Returns the [`Instant`] this clock was created.
    ///
    /// Usually represents when the app was started.
```

Same goes for the other items, but I don't want to spam you with comments :sweat_smile: